File: FoosballApi/Models/TypeScript/Team.model.ts
import { Player } from 'src/app/shared/models/Player.model'

export class Team {
    
    public id: number = 0;
    public name: string = null;
    public offensPlayer: Player = null;
    public defensePlayer: Player = null;
    public rating: number = 0;

}

File: FoosballApi/Models/TeamModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace FoosballApi.Models
{
    public class TeamModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public PlayerModel OffensPlayer { get; set; }
        public PlayerModel DefensePlayer { get; set; }
        public int Rating { get; set; }
    }
}
